tests/drivers/native/test_cursor.py

```diff
@@ -47,3 +47,16 @@ def test_with_settings_in_execution_options(self):
             dict(rv.context.execution_options), {"settings": {"final": 1}}
         )
         self.assertEqual(len(rv.fetchall()), 1)
+
+    def test_set_query_id(self):
+        import uuid
```

Comment:
Import should be in the top of tile.

Please fix flake8 as well.

Reply:
Ran flake8 on the wrong directory :sweat: 
Should be fixed now